internal/clients/tenant.go

```diff
@@ -29,6 +31,12 @@ func NewTenant(tenantServiceURL string, authToken string) Tenant {
 	}
 }
 
+// TenantService is contains methods that makes calls to tenant APIs
+type TenantService interface {
+	GetTenantInfo(tenantID string) (ti TenantInfo, err error)
```

Comment:
nitpick: can remove `ti` and `err`

Reply:
This is as per old implementation. If we are to do this, let's do it with next PR.

Reply:
do you mean we need a new PR for  removing those vars `ti` and `err` ? 
i.e. to make it `GetTenantInfo(tenantID string) (TenantInfo, error)` ?